spec/features/add_a_review_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+feature 'A user can add a review to a Dev Tool' do
+  let!(:devtool1) {FactoryGirl.create(:devtool, id: 1)}
+  let!(:review1) {FactoryGirl.create(:review, devtool_id: 1)}
+  context 'An authenticated user can add a review to a dev tool' do
+    before { visit new_devtool_review_path(devtool1.id) }
+    scenario 'The new review page has a title, and body' do
+      expect(page).to have_content("Title")
+      expect(page).to have_content("Body")
+      expect(page).to have_content("Rating")
+    end
```

Comment:
Make sure you are mocking actual user behavior here. for this test, you should be filling in the form fields and expecting what you filled in to appear on the page.
Additionally, title, body and rating are a bit ambiguous. Make this more robust with some sample test data.